lib/files.py

```diff
@@ -91,6 +91,14 @@ def read_zulip_messages_for_topic(
     f.close()
     return messages
 
+def open_read_config(root_directory):
+    outfile = open_outfile(Path(root_directory).resolve(), Path('_config.yml'), 'r+')
```

Comment:
Shouldn't it be `'r'` only since it is for reading only, not reading + writing?

Reply:
I didn't write these parts of the code, I just fixed any remaining issues in code, and squashed commits. I would update them. Thanks for noticing.

Reply:
Thanks for noticing. I have updated it to remove both `r+` and `w+`.